server/tools/waypoint_editor/edit_mission_waypoints.py

```
@@ -0,0 +1,102 @@
+import os
+import sys
+sys.path.append("../..")
+os.environ.setdefault("DJANGO_SETTINGS_MODULE", "server.settings")
+from django.core.wsgi import get_wsgi_application
+get_wsgi_application()
+
+import argparse
+import json
+import auvsi_suas.models as models
+
+
+def active_mission():
+    '''Returns the current active mission - modified from views/mission.py'''
+    missions = models.mission_config.MissionConfig.objects.filter(
+        is_active=True)
+    if len(missions) != 1:
```

Multiple activate missions is an invalid server configuration. I'd raise an exception in that case:

```py
if len(missions) > 1:
    raise ValueError('Multiple active missions: %s' % missions)

return missions[0] if missions else None
```